config.toml

```diff
@@ -131,27 +131,27 @@ url = "/adoptions"
 weight = 1
 
 [[menu.main]]
-name = "Reporting issues"
```

Comment:
Sorry @trishankatdatadog for messing things up. Thank you for understanding. Let me clarify what I have done in `config.toml` file. I've added `contribute` page before `Reporting issue` page, and changed weights of order. Also I've removed previoud `Contribute` page as it only redirects to `python-tuf` repo, and we want detailed contributing page.

Reply:
Don't be sorry, you're helping us! Sorry if I sounded cranky.

One more thing: please don't rewrite history and force-push your changes (except for good reasons) because it makes new changes harder to detect and review. Thanks in advance!

Reply:
Thanks @trishankatdatadog, I will take care of this😊.